vogon/local_settings.py

```diff
@@ -129,7 +129,10 @@
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 
 STATIC_URL = '/static/'
-# STATIC_ROOT = '/diging/wwwnginx/vogon/static/'
+STATIC_ROOT = '/diging/wwwnginx/vogon/static/'
+MEDIA_ROOT = '/home/jasdeep/Documents'
```

Comment:
Don't hard-code this; you should use the ``os`` package to get the current working directory, and then construct the path relative to that.

e.g. 
```python
PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
MEDIA_URL = os.path.join(PROJECT_ROOT, 'media')
```

Otherwise....
![image](https://cloud.githubusercontent.com/assets/3451594/12952660/382434a0-cfe5-11e5-908c-5415b3d8c108.png)